app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
The `people` dictionary is defined as a class attribute, which is fine, but ensure that it is used correctly to store instances of `Person` by their names.